doc/developers/develop.rst

```diff
@@ -410,6 +410,27 @@ inside the** ``__init__`` **method**. All and only the public attributes set by
 fit have a trailing ``_``. As a result the existence of parameters with
 trailing ``_`` is used to check if the estimator has been fitted.
 
+Links to the documentation
+--------------------------
+Estimators include a method ``_get_url_link``, which is used to get the URL
+of the documentation of the estimator. This function is used to prepare the html
+representation of the estimator, e.g., when using the ``display`` function in a
+Jupyter notebook.
+
+By default, the function returns the empty string if the estimator is not a
+scikit-learn estimator. This check is performed by evaluating whether the
+estimator module is the same as the value in the attribute
+``estimator._doc_link_module``, set to ``sklearn`` by default. If an estimator is instantiated
+without changing the value of ``estimator._doc_link_module``, then an empty
+string is returned.
```

Comment:
I think "instantiated" isn't the right word here? I was expecting to read something like "To enable this feature for estimators in your third party library make sure to set `_doc_link_module` to the name of the library. If an estimator's library and `_doc_link_module` do not match an empty string is returned." Basically, I am not sure what instantiating an estimator class has to do with it.

Part of this is explained in the next paragraph